test/absinthe/phase/execution/non_null_test.exs

```diff
@@ -217,6 +240,51 @@ defmodule Absinthe.Phase.Document.Execution.NonNullTest do
   end
 
   describe "lists" do
+    test "non-null list of non-null list of nullable value returns null value" do
```

Comment:
This seems like it should be redundant when `test "deeply nested nullable value inside non-nullable lists can be null"` exists below, but the latter actually passes in master for the same reason `test "deeply nested non-nullable value inside non-nullable lists cannot be null"` fails (I think). 

So I ended up keeping this test just for my own sanity.